### DISCUSSION TITLE
it can support azure? if yes, pls provide config.xml sample

### DISCUSSION THREAD

**Discussion by yanzeng1029** [sine-io/cosbench-sineio#41] [Q&A] [answered]
if yes, pls provide config.xml sample. Thanks

**Comment by kisscelia** [comment, marked as answer]
@yanzeng1029 thank you for your question.
perhaps not.
if you want to support azure, we should develop with the azure SDK as a new bundle.

totally, i've no more time to do this for now, i am sorry.

maybe i can try to do it in the future.
